spec/util/worker_threads_default_printer_spec.rb

```diff
@@ -3,24 +3,41 @@
 require "logstash/util/worker_threads_default_printer"
 
 describe LogStash::Util::WorkerThreadsDefaultPrinter do
-  let(:settings) { {} }
+  let(:settings)  { {} }
   let(:collector) { [] }
 
   subject { described_class.new(settings) }
 
+  before { subject.visit(collector) }
+
```

Comment:
Ok I'll go with that.  BTW, I always try to contexts for variants. i.e. the text starts with `when` e.g.
`context 'when the settings hash has only user content' do`
I will use a `describe` block.